packages/components/alert/src/alert-close.tsx

```diff
@@ -0,0 +1,20 @@
+import { SVGProps, forwardRef } from 'react'
+import { alert } from '@consolelabs/theme'
+import { CloseLine } from '@consolelabs/icons'
+import { useAlertContext } from './context'
+
+export const AlertCloseIcon = forwardRef<
+  SVGSVGElement,
+  SVGProps<SVGSVGElement>
+>((props, ref) => {
+  const { scheme, variant } = useAlertContext()
+  const { className, ...restProps } = props
+
+  return (
+    <CloseLine
```

Comment:
Make sure the icon from the design:

Storybook:
<img width="129" alt="image" src="https://github.com/consolelabs/web-foundation/assets/12707960/4d66ff1c-76c9-47f9-a54f-2d8cbccac021">

Figma:
<img width="116" alt="image" src="https://github.com/consolelabs/web-foundation/assets/12707960/d8a3307b-f565-4322-a6c9-c70d8afbdf63">